src/boot/efi/boot.c

```diff
@@ -618,6 +622,53 @@ static EFI_STATUS reboot_into_firmware(void) {
         assert_not_reached();
 }
 
+static bool entry_is_visible(ConfigEntry *entry, ConfigEntry *visible_group) {
+        assert(entry);
+
+        if (entry->group_entry == NULL)
```

Comment:
as per coding style, when we check for non-NULL or NULL we don't do explicit `== NULL` and `!= NULL` checks, but simply check relying on C's downgrade to bool feature.

Reply:
done